introduction.md

```diff
@@ -5,3 +5,8 @@ This is my introduction for CodeSchool Mastering GitHub course.
 My favourite Code School path, probably Rails for Zombies (from the Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?